.github/workflows/ci.yml

```diff
@@ -6,13 +6,34 @@ env:
   GRADLE_OPTS: -Dhttp.keepAlive=false
 
 jobs:
-  build:
-    name: build
+  generate-test-list:
+    runs-on: ubuntu-latest
```

Comment:
If this is only running on Linux, I think this means the tests below which need "generate-test-list" will not run on Windows?

Reply:
We are only generating list of tasks which is OS-agnostic. It is already used in generating (and running) tasks on windows machines. Please take a look at #2861 or pretty much every current "CI" workflow: [example](https://github.com/opensearch-project/security/actions/runs/5467098686)

Reply:
Hmm, it would appear the internet lied to me haha. Sounds good then!